tests/integration-tests/tests/runtime_bake/test_runtime_bake.py

```diff
@@ -9,39 +9,45 @@
 # or in the "LICENSE.txt" file accompanying this file.
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
-
+import boto3
 import pytest
 from remote_command_executor import RemoteCommandExecutor
 
-from tests.common.schedulers_common import get_scheduler_commands
 from tests.common.utils import retrieve_latest_ami
 
 
-@pytest.mark.skip(reason="Temporarily disable this test")
-@pytest.mark.dimensions("eu-west-3", "c5.xlarge", "alinux2", "torque")
-@pytest.mark.dimensions("us-east-2", "c5.xlarge", "centos7", "sge")
-@pytest.mark.dimensions("us-east-2", "c5.xlarge", "centos8", "sge")
-@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "ubuntu1804", "sge")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "alinux2", "slurm")
+@pytest.mark.dimensions("us-gov-east-1", "c5.xlarge", "ubuntu1804", "slurm")
 @pytest.mark.dimensions("us-gov-west-1", "c5.xlarge", "ubuntu1804", "sge")
-@pytest.mark.dimensions("us-east-1", "m6g.xlarge", "ubuntu1804", "sge")
-@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "slurm")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "centos8", "torque")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "centos7", "sge")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu2004", "torque")
+# @pytest.mark.dimensions("us-east-1", "m6g.xlarge", "alinux2", "slurm")
+# @pytest.mark.dimensions("us-east-1", "m6g.xlarge", "ubuntu1804", "sge")
 @pytest.mark.usefixtures("instance", "scheduler")
-def test_runtime_bake(scheduler, os, region, pcluster_config_reader, clusters_factory, test_datadir, architecture):
+def test_runtime_bake(
+    scheduler, os, region, pcluster_config_reader, clusters_factory, test_datadir, architecture, s3_bucket_factory
+):
     """Test cluster creation with runtime bake."""
-    # remarkable AMIs are not available for ARM yet
-    ami_type = "remarkable" if architecture == "x86_64" else "official"
+    # Remarkable AMIs are not available for ARM yet
+    # Disable centos7,8 remarkable AMIs, because FPGA AMI volume size is not enough and we need to test epel
+    # installation during runtime baking, but FPGA AMI is baked with epel
+    if architecture == "x86_64" and os not in ["centos7", "centos8"]:
+        ami_type = "remarkable"
+    else:
+        ami_type = "official"
+
+    # Create S3 bucket for pre install scripts
```

Comment:
Can you expand this comment to include an explanation of why we're running the pre-install script?